### PR TITLE
Skotl documentation

### PR DESCRIPTION
**Pull request by skotl**
## Proposed change

  Update to the documentation to show how to call the service from a netdaemon app. I'm conscious that the usage
  is probably obvious to Home Assistant old-hands but for someone like me, who is starting cold with HA and 
  netdaemon, calling underlying python services was a new experience. 
  It's obvious once you figure it out so thought I'd shave some time for the next newbie!

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (which adds functionalit)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests
- [x] Documentation only - no code change 

## Additional information

- This PR fixes or closes issue: usability / doc

## Checklist

- [x] The code change is tested and works locally.
- [x] Local tests pass.
- [x] There is no commented out code in this PR.
- [ ] The code has been formatted using Black (`make black`)
- [ ] Tests have been added to verify that the new code works.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,3 +23,16 @@ For now only the folowing platforms are supported:
 2. Install it in HACS
 3. Clear browser cache (no need to restart Home Assistant)
 4. Go to Configuration -> Integrations -> + -> Search for "NetDaemon"
+
+## Use
```
Comment (ludeeus) [inline README.md:27]: Shouldn't this be documented on <https://netdaemon.xyz/> instead of the integration README?
@helto4real ?

**Comment by skotl** [comment]
FWIW, I wondered the same but then the integration feels to me like a stopgap until the functionality is added to the core netdaemon project.

That said, if you’d prefer the usage in the main pages then feel free to reject the PR and I’ll update netdaemon_docs instead.

File: README.md (hunk 1 above)
Comment (ludeeus) [reply README.md:27]: Actually, this is properly documented already <https://netdaemon.xyz/docs/api/api_state#set-state-of-custom-entities>
That is how you should interface with this from your ND code

Comment (skotl) [reply README.md:27]: Ah - that makes more sense when read in its entirety. The first link on `SetState` says that if you want the entries to be persisted then you need to use the integration, but the second link provides more detail saying that the supported domains *are* persisted.
I'll cancel this PR but I think the main docs could be a little clearer - I think it just needs a minor update to the `SetState` page, which I'll propose.
Thanks

**Comment by skotl** [comment]
Cancelling PR as the main netdaemon_docs is probably a better place to have this information

**Pull request closed without merging**